pyrobosim/pyrobosim/utils/knowledge.py

```diff
@@ -31,7 +31,7 @@ def apply_resolution_strategy(world, entity_list, resolution_strategy):
     :rtype: Entity
     """
     if entity_list is None or len(entity_list) == 0:
-        return None
+        raise Exception("Entitity List Cannot Be Null or 0")
```

Comment:
typo

```suggestion
        raise Exception("Entity list cannot be empty or None")
```

but also, why should this be disallowed rather than returning `None` (even if it's not an explicit line)? Did you have a case in mind for this?